### PR TITLE
ARROW-5724: [R] [CI] AppVeyor build should use ccache

### PR DESCRIPTION
**Pull request by nealrichardson**
Compare https://ci.appveyor.com/project/nealrichardson/arrow/builds/25532576 (the first time, which would have primed the cache) to https://ci.appveyor.com/project/nealrichardson/arrow/builds/25533897 (which would have used the cache). Looks like it saves a little bit of time on the build. 

This patch also includes a trivial R namespace fix.

### CODE REVIEW

File: r/man/arrow-package.Rd [hunk 1]
```diff
@@ -22,11 +22,12 @@ Useful links:
 
 }
 \author{
-\strong{Maintainer}: Romain François \email{romain@rstudio.com}
+\strong{Maintainer}: Romain François \email{romain@rstudio.com} (0000-0002-2444-4226)
```
Comment (fsaintjacques) [inline r/man/arrow-package.Rd]: I'm currious, what's the uuid-like for?

Comment (nealrichardson) [reply r/man/arrow-package.Rd]: https://en.wikipedia.org/wiki/ORCID

**Review by pitrou — APPROVED** [review]
+1 from me. Very nice speedup.

**Comment by pitrou** [comment]
AppVeyor build:
https://ci.appveyor.com/project/nealrichardson/arrow/builds/25534739

**Review by fsaintjacques — APPROVED** [review]

**Comment by pitrou** [comment]
Some conflict needs solving now.

**Comment by nealrichardson** [comment]
Rebased, conflict resolved

**Comment by wesm** [comment]
Appveyor build in progress https://ci.appveyor.com/project/nealrichardson/arrow/builds/25536295

**Comment by nealrichardson** [comment]
https://ci.appveyor.com/project/nealrichardson/arrow/builds/25536295 passed; build failures here seem unrelated 🤷‍♂

**Review by wesm — APPROVED** [review]
+1. Travis CI failures here are ARROW-5728 (just merged fix) and ARROW-5732 (not sure yet what's wrong)

**Pull request closed without merging**